src/reducer/filterReducer.jsx

```diff
@@ -0,0 +1,25 @@
+export const filterReducer = (filterState, action) => {
+    const initialState = {
+        sortby:"",
+        category:[],
+        rating: "",
+        range: 200
+    }
+    switch (action.type){
+        case "SORT_BY":
+            return {...filterState, sortby: action.payload}
+        case "CATEGORY":
+            const {category} = filterState
+          return  category.includes(action.payload) ? {...filterState, category: category.filter(category => category!==action.payload)} :
```

Comment:
I like this, the way have used same type for both checking and unchecking, gonna implement this in my project as well